.github/workflows/gke_e2e_tests.yaml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2022 the Kubeapps contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: "GKE e2e Tests"
+on:
+  workflow_call:
+    inputs:
+      CHARTMUSEUM_VERSION:
+        type: string
+      DEBUG_MODE:
+        type: boolean
+        required: false
+        default: false
+      DEV_MODE:
+        type: boolean
+        required: false
+        default: false
+      #TODO: right now it contains just the version number (eg. 1.22), rename to GKE_VERSION once CircleCI is decommissioned
+      GKE_BRANCH:
+        type: string
+        description: "The branch of the GKE environment in which we want to test."
+      GKE_CLUSTER:
+        type: string
+      GKE_PROJECT:
+        type: string
+      GKE_RELEASE_CHANNEL:
+        type: string
```

Comment:
I'm not getting this part, I think. `GKE_BRANCH` would be either `stable` or `regular`, wouldn't it? If so, what's the difference with `GKE_RELEASE_CHANNEL` ?
Why can't we just use the `GKE_VERSION`  now? I mean, once the name collision is fixed, why can't we have two 1.22 clusters?  (besides, in https://github.com/vmware-tanzu/kubeapps/pull/5567 I bumped it up to 1.23)

Reply:
`GKE_BRANCH` is how it's been called through the code so far, it takes the value from `GKE_STABLE_VERSION` and `GKE_REGULAR_VERSION` (the numeric values, eg `1.22`). Taking into account that those values could be the same (as they've been lately), we're adding the new variable `GKE_RELEASE_CHANNEL` that can take the values `STABLE` and `REGULAR`, to disambiguate them.